CHANGELOG.md

```diff
@@ -32,6 +32,9 @@ please take a look at related PRs and issues and see if the change affects you.
 
 ### Changed
 
+- Added `def_file_name` atribute to `RefRulePosition` for storing the definition's
+  model file name in case of cross-references between models. ([#277])
```

Comment:
Minor: you need to add the reference at the end of the file.

Reply:
Should I also reference the pull request?

Reply:
Yes. This would be great.